.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-rockspec-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: queue-scm-1.rockspec
+
+  publish-rockspec-tag:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Create a rockspec for the release.
+      - run: printf '%s=%s\n' TAG "${GITHUB_REF##*/}" >> "${GITHUB_ENV}"
+      - run: sed -E
+          -e 's/branch = ".+"/tag = "${{ env.TAG }}"/g'
+          -e 's/version = ".+"/version = "${{ env.TAG }}-1"/g'
+          queue-scm-1.rockspec > queue-${{ env.TAG }}-1.rockspec
```

Comment:
It'll not work, because single quotes are used in the rockspec. Please, run this command locally and check result.

Reply:
https://github.com/tarantool/tuple-keydef/blob/946466de4a6044e0bfba30eae851206ce0ae572c/.github/workflows/publish.yml#L25-L30

Reply:
Updated.